locale/loris.pot

```diff
@@ -162,12 +162,24 @@ msgstr ""
 msgid "Sex"
 msgstr ""
 
+msgid "Male"
+msgstr ""
```

Comment:
I'm not sure if we should include these since they're customizable values that come from the db now. We might need to consider how we want to deal with them later.